images/win/scripts/Installers/Configure-Toolset.ps1

```diff
@@ -0,0 +1,84 @@
+################################################################################
+##  File:  Configure-Toolset.ps1
+##  Team:  CI-Build
+##  Desc:  Configure Toolset
+################################################################################
+
+Function Set-DefaultVariables
+{
+    param
+    (
+        [Parameter(Mandatory=$true)]
+        [object] $EnvVars,
+        [Parameter(Mandatory=$true)]
+        [string] $ToolVersionPath
+    )
+
+    $templates = $EnvVars.pathTemplates
+    foreach ($template in $templates)
+    {
+        $toolSystemPath = $template -f $ToolVersionPath
+        Add-MachinePathItem -PathItem $toolSystemPath | Out-Null
+    }
+
+    if (-not ([string]::IsNullOrEmpty($EnvVars.rootVariableName)))
+    {
+        setx $toolEnvVars.rootVariableName $ToolVersionPath /M | Out-Null
+    }
+}
+
+$ErrorActionPreference = "Stop"
+
+Import-Module -Name ImageHelpers -Force -DisableNameChecking
+
+# Define executables for cached tools
+$toolsEnvironmentVariables = @{
+    Python = @{
+        pathTemplates = @(
+            "{0}",
+            "{0}\Scripts"
+        )
+    }
+    Boost = @{
+        variableTemplate = "BOOST_ROOT_{0}_{1}_{2}"
+    }
+    go = @{
+        pathTemplates = @(
+            "{0}\bin"
+        )
+        rootVariableName = "GOROOT"
```

Comment:
May be `defaultVariable` to be consistent with Ubuntu.

Reply:
Done